examples/USGS/USGS_NLCD_RELEASES_2019_REL_NALCMS.js

```diff
@@ -0,0 +1,30 @@
+// Import the NALCMS collection.
+var image = ee.Image('projects/ee-nalcms/assets/nalcms/2020');

```

Comment:
Use the final locatioon:

```
var image = ee.Image('USGS/NLCD_RELEASES/2019_REL/NALCMS');
```

Reply:
Updated this (on all 3 PR's). But can you clarify this for me to understand? When I pull this into GEE and run it, it errors. It only works when I use the location that the assets are in (projects/ee-rcmap, etc). Not quite making sense to me yet. Thanks! @schwehr

Reply:
Please change the path back to the public path. It will not work until we do some changes internally, so it is expected that it fails right now.

Reply:
Aha! I see I'm not following how github shows which code is at the head of the PR. You've got the public location in the script there and that's correct.  Resolving this